src/Clustering/View/GMUDefaultClusterRenderer.h

```diff
@@ -61,6 +61,18 @@ NS_ASSUME_NONNULL_BEGIN
            clusterIconGenerator:(id<GMUClusterIconGenerator>)iconGenerator;
 
 /**
+ * Allows setting a zIndex value for the clusters.  This becomes useful
+ * when using multiple cluster data sets on the map and require a predictable
+ * way of displaying multiple sets with a predictable layering order.
+ *
+ * If no specific zIndex is not specified during the initialization, the 
+ * default zIndex is '1'.  Larger zIndex values are drawn over lower ones
+ * similar to the zIndex value of GMSMarkers.
+ */
+- (instancetype)initWithMapView:(GMSMapView *)mapView
```

Comment:
I think since zIndex is optional it is better to have it as a property (default to 1.0) rather than a new init method.